src/sentry/api/helpers/group_index.py

```diff
@@ -420,6 +420,29 @@ def self_subscribe_and_assign_issue(acting_user, group):
             return Actor(type=User, id=acting_user.id)
 
 
+def track_update_groups(function):
+    def wrapper(request, projects, *args, **kwargs):
+        try:
+            response = function(request, projects, *args, **kwargs)
+        except Exception as error:
```

Comment:
```suggestion
        except Exception:
```
You don't need the variable if you reraise.

Reply:
huh TIL 🙏 thanks!